common/app/controllers/EmailSignupController.scala

```diff
@@ -44,18 +44,22 @@ case class EmailForm(
   }
 }
 
-class EmailFormService(wsClient: WSClient) extends LazyLogging {
+class EmailFormService(wsClient: WSClient) extends LazyLogging with RemoteAddress {
 
-  def submit(form: EmailForm): Future[WSResponse] = if (form.isLikelyBotSubmission) {
+  def submit(form: EmailForm)(implicit request: Request[AnyContent]): Future[WSResponse] = if (form.isLikelyBotSubmission) {
     Future.failed(new IllegalAccessException("Form was likely submitted by a bot."))
   } else {
     val idAccessClientToken = Configuration.id.apiClientToken
     val consentMailerUrl = s"${Configuration.id.apiRoot}/consent-email"
     val consentMailerPayload = JsObject(Json.obj("email" -> form.email, "set-lists" -> List(form.listName)).fields)
+    val headers = clientIp(request)
+      .map(ip => List("X-Forwarded-For" -> ip))
+      .getOrElse(List.empty).:+("X-GU-ID-Client-Access-Token" -> s"Bearer $idAccessClientToken")
```

Comment:
I think we worked out that you could also do infix notation.

`.getOrElse(List.empty) :+ "X-GU-ID-Client-Access-Token" -> s"Bearer $idAccessClientToken"`